.github/workflows/test.yml

```diff
@@ -224,7 +224,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v3
         with:
-          go-version: 1.17
+          go-version: 1.18
       - name: Display go version
         run: go version
       - name: Install runsim
```

Comment:
The line below (l.231) must as well be changed with `go install`.
Maybe in the `release-sims` gh action as well for consistency?